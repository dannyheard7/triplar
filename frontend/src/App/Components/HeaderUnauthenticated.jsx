import React from "react";
import {Link} from 'react-router-dom';

export default function HeaderUnauthenticated(props) {
    return (
        <div className="header">
            <Link to='/' activeclassname="active" className="brand">Triplar</Link>

            <button className="navbar-toggle" type="button" data-toggle="collapse"
                    data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent"
                    aria-expanded="false" aria-label="Toggle navigation">
                <span className="navbar-toggler-icon"></span>
            </button>

            <nav className="navbar">
                <span className="nav-item" id="login-nav-item">
                    <Link to='/login' className="nav-link">Login</Link>
                </span>
                <span className="nav-item" id="register-nav-item">
                    <Link to='/register' className="nav-link">Register</Link>
                </span>
            </nav>
        </div>
    );
}
