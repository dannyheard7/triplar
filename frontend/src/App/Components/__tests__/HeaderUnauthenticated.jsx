import React from "react";
import ShallowRenderer from 'react-test-renderer/shallow';
import HeaderUnauthenticated from "../HeaderUnauthenticated";


const renderer = new ShallowRenderer();


describe('<HeaderUnauthenticated />', () => {

    test('renders correctly', () => {
        const result = renderer.render(<HeaderUnauthenticated  />);
        expect(result).toMatchSnapshot();
    });
});