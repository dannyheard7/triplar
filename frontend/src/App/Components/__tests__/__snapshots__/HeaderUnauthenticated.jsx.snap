// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HeaderUnauthenticated /> renders correctly 1`] = `
<div
  className="header"
>
  <Link
    activeclassname="active"
    className="brand"
    replace={false}
    to="/"
  >
    Triplar
  </Link>
  <button
    aria-controls="navbarSupportedContent"
    aria-expanded="false"
    aria-label="Toggle navigation"
    className="navbar-toggle"
    data-target="#navbarSupportedContent"
    data-toggle="collapse"
    type="button"
  >
    <span
      className="navbar-toggler-icon"
    />
  </button>
  <nav
    className="navbar"
  >
    <span
      className="nav-item"
      id="login-nav-item"
    >
      <Link
        className="nav-link"
        replace={false}
        to="/login"
      >
        Login
      </Link>
    </span>
    <span
      className="nav-item"
      id="register-nav-item"
    >
      <Link
        className="nav-link"
        replace={false}
        to="/register"
      >
        Register
      </Link>
    </span>
  </nav>
</div>
`;
